openslides/motions/static/js/motions/base.js

```diff
@@ -276,9 +276,8 @@ angular.module('OpenSlidesApp.motions', [
                 getListOfSpeakersTitle: function () {
                     var title = gettextCatalog.getString('Motion');
                     if (this.identifier) {
-                        title += ' ' + this.identifier + ':';
+                        title += ' ' + this.identifier;
                     }
-                    title += ' ' + this.getTitle();
```

Comment:
If a motion does not have an identifier, the resoult would be `Motion`. We should return the motion title, if no identifier is given.